ROADMAP.md

```diff
@@ -243,6 +244,21 @@ ping-pong).
 
 - [Protocol Select specification](https://github.com/libp2p/specs/pull/349)
 
+### 🧪 Automated compatibility testing
+
+**What?** There are more than 6 implementations of the [libp2p specification] in
+different languages. We need to ensure compatibility of all combination of these
+implementations. Given the number of libp2p implementations and the amount of
+libp2p protocols, verifying compatibility on a continuous basis can only be
+tackled in an automated fashion.
+
+**Why?** Automated compatibility testing allows us to move fast with confidence
+and will increase trust in the libp2p project as a whole.
```

Comment:
And move some tests currently at the IPFS layer to where they actually belong.

Reply:
Agreed that we should do that, though I would consider it a detail. Do you feel strongly on mentioning it in our roadmap?

Reply:
No, that was just a general comment.